packages/hash/integration/src/tests/model/knowledge/workspaceInstance.model.test.ts

```diff
@@ -0,0 +1,97 @@
+import { getRequiredEnv } from "@hashintel/hash-backend-utils/environment";
+import { createGraphClient } from "@hashintel/hash-api/src/graph";
+import {
+  ensureWorkspaceTypesExist,
+  WORKSPACE_TYPES,
+} from "@hashintel/hash-api/src/graph/workspace-types";
+import { ensureWorkspaceKnowledgeExists } from "@hashintel/hash-api/src/graph/workspace-knowledge";
+import { Logger } from "@hashintel/hash-backend-utils/logger";
+import {
+  UserModel,
+  WorkspaceInstanceModel,
+} from "@hashintel/hash-api/src/model";
+import { workspaceAccountId } from "@hashintel/hash-api/src/model/util";
+import { createTestUser } from "../../util";
+
+jest.setTimeout(60000);
+
+const logger = new Logger({
+  mode: "dev",
+  level: "debug",
+  serviceName: "org-membership-tests",
```

Comment:
I think we are slightly inconsistent with the `serviceName` in the different tests. We should probably just stick to 
```suggestion
  serviceName: "integration-tests",
```

Reply:
Have addressed this and the remainder of inconstant logger service names in https://github.com/hashintel/hash/pull/1327/commits/cd7597dd7db16e6116d041b08bf2bf97e8b70653